static/calculate_score.html

```diff
@@ -325,13 +325,6 @@ <h2><br>Results</h2>
     </div>
   </div>
   <footer class="text-center" style="background-color: #343a40;">
-    <div class="container">
```

Comment:
The copyright is still on the following pages:
index.html
studies.html
cli_download.html
visualize.html

Reply:
I still like the way the dark gray signals the end of the page. For each of the html pages, instead of deleting the whole footer, just remove the text. This leaves a small dark grey footer:
<div class="container">
  <div class="row">
    <div class="col-12 text-white" style="margin-top: 1rem;">
    </div>
  </div>
</div>

Reply:
I made the changes. Let me know what you think.

Reply:
Awesome, thanks! I'm working on the other changes now.